apps/arweave/include/ar_config.hrl

```diff
@@ -67,6 +67,7 @@
 -define(MAX_PARALLEL_POST_CHUNK_REQUESTS, 100).
 -define(MAX_PARALLEL_GET_SYNC_RECORD_REQUESTS, 10).
 -define(MAX_PARALLEL_REWARD_HISTORY_REQUESTS, 1).
+-define(MAX_PARALLEL_GET_TX_REQUESTS, 1).
```

Comment:
What do you think about setting the default a little higher? It looks like our nodes that were underload from `GET /tx/{hash}` or `GET /tx/{hash}/status` requests were getting hit with like 50 per second. But the ones that weren't falling over could still get hit with up to 5 or 10 requests per second in short bursts. The latency on those requests was pretty short so maybe it doesn't really matter.

What about a default of 5 or 10? Do you think bumping it to 10 would make the full-chain nodes fallover?

Reply:
(PS: I changed the PR base to `release/performance-2.8`)